tests/integration/base_object_test.go

```diff
@@ -122,6 +122,7 @@ func runObjectE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite
 	dosuErr := helper.ObjectUserClient.Delete(namespace, userid)
 	require.Nil(s.T(), dosuErr)
 	logger.Infof("Object store user deleted successfully")
+	require.False(s.T(), helper.ObjectUserClient.UserSecretExists(namespace, userid, storeName), "make sure user secret was deleted")
```

Comment:
The integration tests are failing this new test. It's likely because there is a delay between deleting the crd and the deletion of the secret. The test would need to retry a couple times while you wait for the secret to be deleted.

Reply:
I could add a 5 second sleep like there is between the ObjectStoreUser create and the check for the secret existing. 
https://github.com/rook/rook/blob/master/tests/integration/base_object_test.go#L59

Reply:
@colonwq Yes, that should be sufficient.